spec/MacAddressTableRecorder.yaml

```diff
@@ -2997,78 +3142,78 @@ paths:
             backend-time:
               schema:
                 type: integer
                 example: 850
-              description: 'Value written by the service provider, reporting the elapsed time for data retrieval from the backend (service invocation, database access…). Expressed in milliseconds'
+              description: "Value written by the service provider, reporting the elapsed time for data retrieval from the backend (service invocation, database access…). Expressed in milliseconds"
             life-cycle-state:
               schema:
                 type: string
                 enum:
-                  - 'EXPERIMENTAL'
-                  - 'OPERATIONAL'
-                  - 'DEPRECATED'
-                  - 'OBSOLETE'
-                  - 'UNKNOWN'
-                  - 'NOT_YET_DEFINED'
-                example: 'EXPERIMENTAL'
+                  - "EXPERIMENTAL"
+                  - "OPERATIONAL"
+                  - "DEPRECATED"
+                  - "OBSOLETE"
+                  - "UNKNOWN"
+                  - "NOT_YET_DEFINED"
+                example: "EXPERIMENTAL"
               description: >
-                  'Life cycle state of the consumed service
-                  find in [/core-model-1-4:control-construct/logical-termination-point=matr-1-0-0-op-s-bm-004/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-configuration/life-cycle-state]'
-        '400':
-          $ref: '#/components/responses/responseForErroredServiceRequests'
-        '401':
-          $ref: '#/components/responses/responseForErroredServiceRequests'
-        '403':
-          $ref: '#/components/responses/responseForErroredServiceRequests'
-        '404':
-          $ref: '#/components/responses/responseForErroredServiceRequests'
-        '500':
-          $ref: '#/components/responses/responseForErroredServiceRequests'
+                'Life cycle state of the consumed service
+                find in [/core-model-1-4:control-construct/logical-termination-point=matr-1-0-2-op-s-bm-004/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-configuration/life-cycle-state]'
+        "400":
+          $ref: "#/components/responses/responseForErroredServiceRequests"
+        "401":
+          $ref: "#/components/responses/responseForErroredServiceRequests"
+        "403":
+          $ref: "#/components/responses/responseForErroredServiceRequests"
+        "404":
+          $ref: "#/components/responses/responseForErroredServiceRequests"
+        "500":
+          $ref: "#/components/responses/responseForErroredServiceRequests"
         default:
-          $ref: '#/components/responses/responseForErroredServiceRequests'
+          $ref: "#/components/responses/responseForErroredServiceRequests"
   /v1/inquire-oam-request-approvals:
```

Comment:
The following description shall be added to this path , 
```
description: >
      'DEPRECATED
      Service shall be deleted with AP v3.0.0
      Starting from AP v2.1.0, /v1/inquire-basic-auth-approvals shall be applied as a backward compatible update'
```

Reply:
fixed